pkg/apis/serving/k8s_validation.go

```diff
@@ -318,8 +318,9 @@ func validateContainerPorts(ports []corev1.ContainerPort) *apis.FieldError {
 	// Don't allow userPort to conflict with QueueProxy sidecar
 	if userPort.ContainerPort == networking.BackendHTTPPort ||
 		userPort.ContainerPort == networking.BackendHTTP2Port ||
-		userPort.ContainerPort == networking.RequestQueueAdminPort ||
-		userPort.ContainerPort == networking.RequestQueueMetricsPort {
+		userPort.ContainerPort == networking.QueueAdminPort ||
```

Comment:
Please update the spec.md that this port is taken. Also we need that "block of ports" bug, I guess, while I remember it :)

Reply:
Spec.md is modified and issue created (#4123)